pallets/liquidity-pools/src/lib.rs

```diff
@@ -426,7 +426,7 @@ pub mod pallet {
 			// TODO(future): Once we diverge from 1-to-1 conversions for foreign and pool
 			// currencies, this price must be first converted into the currency_id and then
 			// re-denominated to 18 decimals (i.e. `Ratio` precision)
-			let price_value = T::TrancheTokenPrice::get(pool_id, tranche_id)
+			let (price, computed_at) = T::TrancheTokenPrice::get_price(pool_id, tranche_id)
```

Comment:
@lemunozm can we in the same run also close this todo and computed the conversion?

Reply:
I'm not sure if it's super straightforward. Do we need to add the order-book dependency to know the market price used for foreign/pool currencies?

Reply:
I agree it isn't. But we are not only touching the UTs here but also chaning the return types, naming etc. Then we could also close the todo IMO.

Reply:
Done here: [15424a8](https://github.com/centrifuge/centrifuge-chain/pull/1890/commits/15424a8ea2d379f164ac44d750af2574d6d0af8f) @mustermeiszer